regression-test/suites/ddl_p0/test_create_table_without_distribution.groovy

```diff
@@ -37,7 +38,6 @@ suite("test_create_table_without_distribution") {
     insert into test_create_table_without_distribution values(1,2);
     """
     qt_test_select_old_planner "select * from test_create_table_without_distribution;"
-    qt_test_show_old_planner " show create table test_create_table_without_distribution;"
-
-
+    def res2 = sql "show create table test_create_table_without_distribution;"
+    assertEquals(res2[0][1].contains("DISTRIBUTED BY RANDOM BUCKETS AUTO"), true)
```

Comment:
```suggestion
    logger.info(res2[0][1])
    assertTrue(res2[0][1].contains("DISTRIBUTED BY RANDOM BUCKETS AUTO"))
```

Reply:
add  `assertContains` method to plugin, and print the return result to log.

for example
```groovy
assertContains(res2.toString(), "DISTRIBUTED BY RANDOM BUCKETS AUTO")
```

plugin example: https://github.com/apache/doris/blob/master/regression-test/plugins/plugin_check.groovy